### PR TITLE
Update exchange.graphql

### PR DESCRIPTION
**Pull request by ashkan18**
# Change
Exchange was updated to add two new timestamps on order in https://github.com/artsy/exchange/pull/51, this PR follows the update and exposes those in MP.

cc: @sepans

### CODE REVIEW

File: src/schema/ecommerce/create_order_mutation.js [hunk 1]
```diff
@@ -87,6 +87,8 @@ export const CreateOrderMutation = mutationWithClientMutationId({
             createdAt
             updatedAt
             createdAt
```
Comment (williardx) [inline src/schema/ecommerce/create_order_mutation.js]: Not related to this PR but should `createdAt` be in here twice?

Comment (sepans) [reply src/schema/ecommerce/create_order_mutation.js]: Probably a merge thing. Removed it in another PR. Thanks for catching it.

Comment (williardx) [reply src/schema/ecommerce/create_order_mutation.js]: Thanks @sepans!

**Comment by williardx** [comment]
@ashkan18 Looks good but there are some merge conflicts 😢

**Comment by ashkan18** [comment]
merge on green

**Pull request merged**